plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeMinioWithExclusiveWritesAndHmsConnectorSmokeTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.testing.containers.Minio.MINIO_ACCESS_KEY;
+import static io.trino.testing.containers.Minio.MINIO_REGION;
+import static io.trino.testing.containers.Minio.MINIO_SECRET_KEY;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Delta Lake connector smoke test exercising Hive metastore and MinIO storage with exclusive writes enabled
+ */
+public class TestDeltaLakeMinioWithExclusiveWritesAndHmsConnectorSmokeTest
+        extends BaseDeltaLakeAwsConnectorSmokeTest
+{
+    @Override
+    protected Map<String, String> hiveStorageConfiguration()
+    {
+        return ImmutableMap.<String, String>builder()
+                .put("fs.hadoop.enabled", "false")
+                .put("fs.native-s3.enabled", "true")
+                .put("s3.aws-access-key", MINIO_ACCESS_KEY)
+                .put("s3.aws-secret-key", MINIO_SECRET_KEY)
+                .put("s3.region", MINIO_REGION)
+                .put("s3.endpoint", hiveMinioDataLake.getMinio().getMinioAddress())
+                .put("s3.path-style-access", "true")
+                .put("s3.exclusive-create", "true")
+                .put("s3.max-connections", "2")
+                .buildOrThrow();
+    }
+
+    @Override
+    protected Map<String, String> deltaStorageConfiguration()
+    {
+        return ImmutableMap.<String, String>builder()
+                .put("fs.hadoop.enabled", "false")
+                .put("fs.native-s3.enabled", "true")
+                .put("s3.aws-access-key", MINIO_ACCESS_KEY)
+                .put("s3.aws-secret-key", MINIO_SECRET_KEY)
+                .put("s3.region", MINIO_REGION)
+                .put("s3.endpoint", hiveMinioDataLake.getMinio().getMinioAddress())
+                .put("s3.path-style-access", "true")
+                .put("s3.exclusive-create", "true")
+                .put("s3.streaming.part-size", "5MB") // minimize memory usage
+                .put("s3.max-connections", "4") // verify no leaks
+                .buildOrThrow();
+    }
+
+    @Test
+    public void testDeltaColumnInvariant()
```

Comment:
can this class simply override TestDeltaLakeMinioAndHmsConnectorSmokeTest? are these tests any different from what we have in TestDeltaLakeMinioAndHmsConnectorSmokeTest?

Reply:
Not really, TestDeltaLakeMinioAndHmsConnectorSmokeTest disables some tests that we actually want to run because they will now work properly